tests/generic/_generic.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+
+marcos:
```

Comment:
❓ I could not find anything about documenting tests? In our docs page they are referred as macros, so I assume this should work, but it doesn't :-/

Reply:
I think this will work when you fix the typo of `marcos` -> `macros` :)

Reply:
🤦